pkg/kubelet/images/image_manager.go

```diff
@@ -60,10 +60,6 @@ func NewImageManager(recorder record.EventRecorder, imageService kubecontainer.I
 // shouldPullImage returns whether we should pull an image according to
 // the presence and pull policy of the image.
 func shouldPullImage(container *v1.Container, imagePresent bool) bool {
-	if container.ImagePullPolicy == v1.PullNever {
```

Comment:
I actually like the explicit check. If we add more pull policy one day, this would still work correctly.

Reply:
Even if we add more policy, we only need to judge when the function returns true. and all others returns false immediatly. We do not need to judge when this function returns false additionally. I think there is no problem if we delete this check.